spec/factories/authorizations.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :authorization do
+    provider 'github'
```

Comment:
@sinpai let's get `github` or `google` randomly

Reply:
👍 But I wouldn't change `Faker::Omniauth.github['uid']` as uid will be applicable both to google and github